examples/with-tailwind/packages/ui/tsup.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig, Options } from "tsup";
+
+export default defineConfig((options: Options) => ({
+  treeshake: true,
+  splitting: true,
+  entry: ["src/**/*.tsx"],
+  format: ["esm"],
+  dts: true,
+  minify: true,
+  clean: true,
+  external: ["react", "react-dom"],
```

Comment:
ooooof.

As a JS community we need to do a better job of dealing with dynamically linked libraries rather than just enumerating them _literally everywhere_.

That we do all compilation in isolation without a linking pass is killing me.